docs/modules/ROOT/pages/general-info/supported-platforms.adoc

```diff
@@ -6,20 +6,24 @@ details the JVMs currently supported by the Payara Platform Community (both Paya
 
 == Supported Java Virtual Machines
 
-The Payara Platform currently supports the following Java Virtual Machines:
+The Payara Platform currently runs the following Java Virtual Machines:
 
-* Oracle JDK8 (u162+), Oracle JDK 11 (11.0.5+)
-* Azul Zulu JDK8 (u162+), Azul Zulu JDK 11 (11.0.5u10+)
-* OpenJDK JDK8 (u162+), OpenJDK 11 (11.0.5+)
-* All other JVMs based on OpenJDK 8u162+ or 11.0.5+ (for example Corretto JDK, Liberica JDK, AdoptOpenJDK)
+* Azul Zulu JDK: 8 (u162+), 11 (11.0.5u10+)
+* Oracle JDK: 8 (u162+), 11 (11.0.5+)
+* Amazon Corretto: 8, 11 (11.0.5+)
+* Adopt Open JDK: 8, 11 (11.0.5+)
+* Adopt Open JDK with Eclipse Open J9
+* Any other JVM based on OpenJDK 8u162+ or 11.0.5+
 
-TLS 1.3 is supported on JDK 8 with Azul Zulu 1.8.222+, Open JDK 8u262+ based builds, and all JDK 11 versions.
+TLS 1.3 is supported on JDK 8 with Azul Zulu 1.8.222+, Open JDK 8u262+ based builds, and all version 11 of all above JDKs.
```

Comment:
I assume the 8u262+ builds of OpenJ9 and Corretto also support TLS 1.3 (as it is in openJDK)

Reply:
I assume so, I didn't check it out. I think it's safe to assume so, since they are based on 8u262.